configuration/pipeline_config.json

```diff
@@ -0,0 +1,69 @@
+{
+  "PipelineName": "IMAQAL_COVID19",
+  "RawDataSources": [
+    {
+      "SourceType": "RapidPro",
+      "Domain": "textit.in",
+      "TokenFileURL": "gs://avf-credentials/imaqal-text-it-token.txt",
+      "ContactsFileName": "imaqal_covid19_contacts",
+      "ActivationFlowNames": [
+        "imaqal_mag_c19_s01e01_activation"
```

Comment:
If it's not too late, can this be covid19 instead of c19?

Reply:
We are still in dev mode. updated to imaqal_covid19_mag_s01e01_activation